### ISSUE TITLE
[Components and pattern standards] The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA. (04.01.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30428]
#### VFS team name
Public Websites

#### VFS product name
ORDMI Harassment Reporting Tool - Landing Page

#### Point of Contact/Reviewers
Mikki Northuis (IA)

---

### Platform Issue
The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA.

### Issue Details
The breadcrumb is not accurate - it must represent the current page, and all parent pages should be clickable links.  The breadcrumb for this page should be "Home  >  How to report harassment at VA", where "Home" is clickable and links to VA.gov home, and "How to report harassment at VA" is bolded and not interative. 

### VA.gov Experience Standard
[Cateogy Number 04, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4 A

---

### Platform Recommendation
Correct breadcrumb. 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by mnorthuis** [comment]
This will need to be reviewed once the page is published.  Removing the launch-blocking label.

**Comment by DanielleThierryUSDSVA** [comment]
Will validate after publishing.

**Comment by laflannery** [comment]
Confirmed that that this page has been published and the breadcrumbs are displayed as described in the ticket description:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/106678594/8daee360-bd53-4279-8c65-5eff29fab253)
